src/pages/workspaces/List.ts

```diff
@@ -0,0 +1,10 @@
+import { WorkspacesList } from 'src/pages/workspaces/WorkspacesList/WorkspacesList';
```

Comment:
I *think* we want `WorkspacesList` in `src/workspace`. Is that correct, @msilva-broad ? But then this wrapper file remains in `src/pages/worksapces`?

Reply:
correct!

and I love that this split was done.  All our page roots should have this split to keep the nav stuff in a wrapper-layer above the main component.